network-api/networkapi/templates/reviewapp-help.html

```diff
@@ -13,15 +13,16 @@ <h1 class="h1-heading">Review app help page</h1>
         </div>
         <p>Test pages with placeholder content:</p>
         <ul>
-          <li><a href="/about/">About</a></li>
-          <li><a href="/style-guide/">Style guide</a></li>
-          <li><a href="/news/">News</a></li>
-          <li><a href="/initiatives/">Initiatives</a></li>
-          <li><a href="/initiatives/single-page-opportunity/">Single page opportunity</a></li>
-          <li><a href="/initiatives/multi-page-opportunity/">Multi page opportunity</a></li>
+          <li><a href="/who-we-are/">Who we are</a></li>
+          <li><a href="/what-we-do/">What we do</a></li>
+          <li><a href="/what-you-can-do/">What you can do</a></li>
+          <li><a href="/Blog/">Blog</a></li>
```

Comment:
Uppercase b in `/Blog/`, is that an issue?

Reply:
not a true "issue" but good to keep lowercase.

Reply:
```suggestion
          <li><a href="/blog/">Blog</a></li>
```